corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
     visibility:
       type: string
       enum: [public,private]
+    project:
```

Comment:
Is there any reason to omit `needs_attestation`, `status`, `tc_uri`, `processing_state` or `validation_state`?

Reply:
I wasn't sure if the was need from a get project. I'm happy to add them in.

Reply:
the needs attestations seems like something that would be returned to the submitter, not someone viewing the project, same with status, processing_state, and validations_state. 

Also what is the tc_uri?

Reply:
So the response body for `GET /v1/submission/{project_uuid}` is also missing still and I was thinking that you'd use the same project component as a return object for both `GET /v1/submission/{project_uuid}` and `GET /v1/project/{project_uuid}` to keep it simple, in which case you need all of the above pieces of metadata.

Did you have an alternative idea though/were you planning on creating a different object for the  `GET /v1/submission/{project_uuid}` API call?

`tc_uri` is the S3 URI designating the location of the project's terms and conditions form that we need to store.

Reply:
I'll recycle it for both.